coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java

```diff
@@ -354,11 +354,11 @@ public void testSchemaPromotionView() {
   @Test
   public void testUnionExtractUDF() {
     RelNode relNode = TestUtils.toRelNode("SELECT extract_union(foo) from union_table");
-    String targetSql = String.join("\n", "SELECT extract_union(foo)", "FROM default.union_table");
+    String targetSql = String.join("\n", "SELECT foo", "FROM default.union_table");
     assertEquals(CoralSpark.create(relNode).getSparkSql(), targetSql);
 
     RelNode relNode2 = TestUtils.toRelNode("SELECT extract_union(foo, 2) from union_table");
-    String targetSql2 = String.join("\n", "SELECT extract_union(foo, 2)", "FROM default.union_table");
+    String targetSql2 = String.join("\n", "SELECT foo.tag_2", "FROM default.union_table");
```

Comment:
Would the test fail if DaliSource starts providing `member` instead of `tag`?

Reply:
It will, but for the extract_union Hive UDF, the original semantics will always be `tag`.